js/adapt-contrib-instructionError.js

```diff
@@ -29,6 +40,44 @@ class InstructionError extends Backbone.Controller {
     notify.popup(notifyObject);
   }
 
+  showInlineError(model) {
+    const data = model.toJSON();
+    this.addErrorClass(model);
+
+    // Update instruction text
+    const dataWithInitial = Object.assign(data, { instruction: model.get('instructionInitial') });
+    const instruction = Handlebars.compile(this.config.body)(dataWithInitial);
+    model.set('instruction', instruction);
+
+    // Focus on instruction text element
+    const $instruction = $(`.${data._id}`).find('.component__instruction').first();
+    a11y.focusFirst($instruction, { defer: true });
```

Comment:
Why did you decide to move the focus to the instruction text on submit click rather than use `aria-live`?

Reply:
I was on the fence. This was mainly due to your note about the visual error potentially being offscreen. So, sighted users may not see it.

Reply:
It will need testing across all the devices as it is.

Reply:
I think this works well moving the focus to the instruction text on submit click rather than using `aria-live`. Reasoning below:
- This uses an existing instruction text element, an `aria-live` attribute should be set on an empty element and injected when an update is made.
- An `aria-live` element is **not** given focus. Depending on the question widget size, whilst focus is on the submit button the instruction text is likely to be off screen for small devices or viewport. I think it makes sense to focus the instruction so both visual and audio instruction exists. 
- Rather than having to tab back through a component from the submit button, focusing on the instruction text is a logical place to resume the question widget.
Note, I referred to [MDN aria-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live#description) for the first two points.

I've tested this on Safari Mac and iPhone (with and without voiceOver running), Chrome and Edge (with and without NVDA running).

Reply:
Thank